src/components/attendee/InputSelect/InputSelect.tsx

```diff
@@ -55,49 +57,27 @@ export const InputSelect: React.FC<InputSelectProps> = ({
 
   const inputClassNames = classNames(styles.input, inputClassName);
 
-  const selectWrapperClassNames = classNames(styles.selectWrapper, {
-    [styles.invalid]: errorSelect,
-  });
-  const selectClassNames = classNames(styles.select, containerClassNames);
-
   return (
     <div className={containerClassNames}>
       <form className={styles.wrapper} autoComplete="off">
-        <div className={wrapperClassNames}>
-          <input
-            className={inputClassNames}
-            placeholder={inputPlaceholder}
-            autoComplete="off"
-            {...register(inputName, inputRules)}
-          />
-          {errorInput && <span className={styles.errorIcon}></span>}
-        </div>
-        <div className={selectWrapperClassNames}>
-          <select
-            className={selectClassNames}
-            {...register(selectName, selectRules)}
-          >
-            <option value={""} disabled>
-              {selectPlaceholder}
-            </option>
-            {Object.entries(urlSources).map(([key, label]) => (
-              <option key={key} value={label}>
-                {label}
-              </option>
-            ))}
-          </select>
-          {errorSelect && (
+        <div className={styles.inputContainer}>
+          <div className={wrapperClassNames}>
+            <input
+              className={inputClassNames}
+              placeholder={inputPlaceholder}
+              autoComplete="off"
+              {...register(inputName, inputRules)}
+            />
+            {errorInput && <span className={styles.errorIcon}></span>}
+          </div>
+          {errorInput && (
             <span className={classNames(styles.error, errorTextClassName)}>
-              {errorSelect?.message}
+              {errorInput?.message}
             </span>
           )}
         </div>
+        <ProfileLinkIcon link={urlValue} index={index} setValue={setValue} />
```

Comment:
Maybe we can improve the `setValue` prop and give it more descriptive name, that way it will be more obvious what it's used for. Ex: `onRender` or something like that.

Reply:
[04f5003](https://github.com/sparkletown/sparkle/pull/3027/commits/04f5003da32c5d46ad3c388886090803250bdac9)